src/components/ThreejsEditorModal.js

```diff
@@ -110,7 +128,29 @@ export class ThreejsEditorModal extends ModalDialog {
         });
     }
 
-    eventToOnHideArgs(e) {return ThreeDSceneDataToMaterial(this.editor.scene.toJSON())}
+    onHide(e) {
+        try {
+            const material = ThreeDSceneDataToMaterial(this.editor.scene);
```

Comment:
I'd move this to a separate function - check material and show alert

Reply:
Done.